examples/quantum_die.py

```diff
@@ -28,47 +28,43 @@ def qubits_needed(n):
     """
     return int(math.ceil(math.log(n, 2)))
 
+def qvm(n):
+    """
+    Connect to a QVM that simulates n sides.
+    """
+    if qubits_needed(n) == 9:
```

Comment:
Is there any benefit of using the `9q-square-qvm` here? Could we use a general QVM always to reduce simplicity in the example?

Reply:
agree